aws/src/vault.js

```diff
@@ -0,0 +1,21 @@
+const http = require('http');
```

Comment:
if possible lets try to write a test case for these new workflows

Reply:
Incorporated the change.